create_derivatives/tests.py

```diff
@@ -118,3 +142,42 @@ def test_run(self, mock_get_object, mock_init):
     def tearDown(self):
         for f in Path(settings.SRC_DIR).iterdir():
             f.unlink()
+
+
+class AWSUploadTestCase(TestCase):
+
+    def setUp(self):
+        """Creates a temp directory and adds an uncompressed bag fixture to the temp directory and database"""
+        path = Path(settings.TMP_DIR)
+        if not path.exists():
+            path.mkdir(parents=True)
+        bag = "3aai9usY3AZzCSFkB3RSQ9"
+        fixture_directory = "aws_upload_bag"
+        bag_path = Path(settings.TMP_DIR, bag)
+        if not Path(bag_path).exists():
+            shutil.copytree(Path("create_derivatives", "fixtures", fixture_directory, bag), bag_path)
```

Comment:
I'm not sure if all of this is necessary - because you're mocking the AWSClient methods I don't think the files are ever touched.

Reply:
While it's not touching the files, it does require the directories to exist, as far as I can tell.